src/NzbDrone.Core/IndexerSearch/MoviesSearchService.cs

```diff
@@ -47,7 +82,9 @@ public void Execute(MoviesSearchCommand message)
 
         public void Execute(MissingMoviesSearchCommand message)
         {
-            var movies = _movieService.MoviesWithoutFiles(new PagingSpec<Movie>
+            List<Movie> movies;
+
+            movies = _movieService.MoviesWithoutFiles(new PagingSpec<Movie>
```

Comment:
You should be able to leave this as `var movies = ...`; functionally it's the same. NBD.